src/Model/Storage/Database.php

```diff
@@ -39,47 +38,57 @@ class Database extends AbstractStorage
 
 	/**
 	 * @param DBA             $dba
-	 * @param LoggerInterface $logger
-	 * @param L10n            $l10n
 	 */
-	public function __construct(DBA $dba, LoggerInterface $logger, L10n $l10n)
```

Comment:
Why did you forgo the logger and the l10n dependencies?

Reply:
I thought the exceptions superseded the log-points. For me there's no need to call a log-point and throw the same exception to the caller.
l10n --> see https://github.com/friendica/friendica/pull/10582#discussion_r686248259